### PR TITLE
Added some event-prefixes, ref #249

### PR DESCRIPTION
**Pull request by sreichel**
- fixes wrong EAV prefixes from #249
- removed block prefixes (wasnt complete)

### CODE REVIEW

**Comment by colinmollenhour** [comment]
Note, while this is a good change it does have some potential to break stuff for people who were using the `core_abstract` prefix before.. It is easy to fix but may not be obvious. Also, the method `clearInstance` uses only the prefixed event so the chances may be slightly higher here since there is only `core_abstract_clear` and not `model_clear`.

Here is an idea.. Add a migration script which checks the config for any observers of the core_abstract prefixed events and if they exist then add an admin notification.

**Comment by sreichel** [comment]
For model save/delete using `core_abstract` should still work , but I wasnt aware of `clearInstance` method.

Closed till migration script is ready.

**Comment by LeeSaferite** [comment]
Please don't get discouraged on this task. I think this is a highly useful PR and we all want to get it merged once it's ready.

**Comment by sreichel** [comment]
No worries, but @colinmollenhour is jut right. I'll reopen soon ;)

**Comment by sreichel** [comment]
Time to re-open :)

**Comment by kiatng** [comment]
@sreichel Why is this closed? May be it's better for branch 20?

**Comment by sreichel** [comment]
@kiatng I've just no interesst to support OpenMage anymore.

**Comment by kiatng** [comment]
@sreichel I'd like to express my deep gratitude for the works you have done to OpenMage. I appreciate your passion and commitment. It's a great loss to the community, so I hope that you can find new interest in OM again in the future.

**Comment by ADDISON74** [comment]
@sreichel - A big loss for OM community. All my respect for your work over the years.

**Comment by ADDISON74** [comment]
@colinmollenhour: This commit should not be closed. I understood Sven did not want to spend time in this project and I respect for his decision. But if someone would like to continue this he should take the benefits of Sven's work.

**Comment by colinmollenhour** [comment]
Agreed, but it still needs some work I think to avoid breaking things (see my first comment).

**Comment by sreichel** [comment]
Even if it is off-topic ... just 2 words to explanation.

I have a problem with Daniel's behavior. Who calls himself maintainer, but largely does not participate in reviews - even those to which he is invited - should perhaps be quieter in other places. PRs to keep them openmage website up to date are not responded to for months. All modules under "ModuleFostering" could have had a modman/composer.json file in the meantime, but that was blocked because I should rather have extended his scripts. There's enough work in that. But then with things, like the allcontributor bot make itself important and scream for RFC does not fit.

Most of the work has been done in recent years by others ...

I continue to follow this and am glad that the PR was opened again.

See you the days (eventually).

**Comment by ADDISON74** [comment]
@sreichel: I haven't been in this project for a long time, but I could see certain things. Many of those who contributed in the beginning and who have the status of administrators are no longer among the posts. I took a look at the list of contributions and and I found your name on the first position. I was rarely allowed to meet a person so devoted to a project. We also had discussions on Stack Exchange and since then you have been focusing on fixing bugs and improvements. When I realized that you closed several PRs on March 17, I searched for them, analyzed them and asked to be reopened. Honestly, it's a shame that your work is wasted. This project cannot survive without the people who have dedicated their time for a purpose. So far I have seen many nice projects that have stopped because those who maintained them have lost interest. It is appreciated that we still have so many people who know Magento 1 very well.

As for Daniel, I believe that an open and honest discussion could solve all the problems. There are indeed many PRs that should be included in OpenMage, but the reviews are very late. In addition, we really need people to test all these changes not to press a button.

**Comment by fballiano** [comment]
@sreichel could you rework this PR based on 1.9.4.x?

**Comment by ADDISON74** [comment]
This PR needs to be rebased to OpenMage:1.9.4.x.

**Comment by sreichel** [comment]
Rebased. 

Btw ... sorry for not doing so much reviews/tests.

I want to finish what i've started ... full IDE-support, make phpstan work ... no complaints from phpcs ...

File: app/code/core/Mage/Catalog/Model/Resource/Collection/Abstract.php [hunk 1]
```diff
@@ -28,6 +28,13 @@
  */
 class Mage_Catalog_Model_Resource_Collection_Abstract extends Mage_Eav_Model_Entity_Collection_Abstract
 {
+    /**
+     * Model event prefix
+     *
+     * @var string
+     */
+    protected $_eventPrefix = 'catalog_resource_collection_abstract';
```
Comment (kiatng) [inline app/code/core/Mage/Catalog/Model/Resource/Collection/Abstract.php:36]: I cannot find `$_eventPrefix` in parent classes. Where is it?

File: app/code/core/Mage/Customer/Model/Resource/Customer/Collection.php [hunk 2]
```diff
@@ -29,6 +29,14 @@
  */
 class Mage_Customer_Model_Resource_Customer_Collection extends Mage_Eav_Model_Entity_Collection_Abstract
 {
+    /**
+     * Model event prefix
+     *
+     * @see Mage_Eav_Model_Entity_Collection_Abstract::$_eventPrefix
+     * @var string
+     */
+    protected $_eventPrefix = 'customer_resource_customer_collection';
```
Comment (kiatng) [inline app/code/core/Mage/Customer/Model/Resource/Customer/Collection.php:38]: This one too, I cannot find reference of `$_eventPrefix` in parent classes.

File: app/code/core/Mage/Customer/Model/Resource/Wishlist/Collection.php [hunk 3]
```diff
@@ -27,6 +27,14 @@
  */
 class Mage_Customer_Model_Resource_Wishlist_Collection extends Mage_Eav_Model_Entity_Collection_Abstract
 {
+    /**
+     * Model event prefix
+     *
+     * @see Mage_Eav_Model_Entity_Collection_Abstract::$_eventPrefix
+     * @var string
+     */
+    protected $_eventPrefix = 'customer_resource_wishlist_collection';
```
Comment (kiatng) [inline app/code/core/Mage/Customer/Model/Resource/Wishlist/Collection.php:36]: Also this one.

**Pull request closed without merging**